fv3core/stencils/updatedzd.py

```diff
@@ -233,6 +232,172 @@ def edge_profile_stencil(
         qe1x, qe2x, qe1y, qe2y = edge_profile_reverse(qe1x, qe2x, qe1y, qe2y, gam)
 
 
+class UpdateDeltaZOnDGrid:
+    """
+    Fortran name is updatedzd.
+    """
+
+    def __init__(self, grid, column_namelist, k_bounds):
+        self.grid = spec.grid
+        self._column_namelist = column_namelist
+        if any(
+            column_namelist["damp_vt"][kstart] <= 1e-5
+            for kstart in range(len(k_bounds))
+        ):
+            raise NotImplementedError("damp <= 1e-5 in column_cols is untested")
+        self._k_bounds = k_bounds  # d_sw.k_bounds()
+        largest_possible_shape = self.grid.domain_shape_full(add=(1, 1, 1))
```

Comment:
we use this for almost every object and we don't usually call it this, will be one of the things we can make consistent in another pass.

Reply:
Alternatively, if we're using the same shape for every storage we use, we could have a single static function `make_storage(origin=None)`. It feels a little weird to pass the same shape in every time we're calling this, and I couldn't come up with a name I'm happy with for the shape (including `largest_possible_shape`).